experimental/Schemes/AffineCurve.jl

```diff
@@ -0,0 +1,42 @@
+export AffineCurve
+export underlying_scheme
+export ambient_space
+
+mutable struct AffineCurve{BRT, BRET, SpecType<:Spec, PolyType<:MPolyElem} <: AbsSpec{BRT, BRET}
+  C::SpecType # the underlying scheme
+  A::SpecType # the ambient space
+  f::Vector{PolyType} # the defining equations
+
+  # Construct the affine curve in Spec(R) which is the zero locus of fⱼ in the hypersurface 
+  # complement of Πᵢ gᵢ.
+  function AffineCurve(R::MPolyRing, f::Vector{PolyType}, g::Vector{PolyType}) where {PolyType<:MPolyElem}
```

Comment:
Sorry, I am bit slow, but why does an affine curve require some `g`? Why not just `f`?

Reply:
It should not require that, indeed. So this should better be an optional argument. The `g` is for the case where also localization at a hypersurface takes place.